scss/_patterns_side-navigation.scss

```diff
@@ -518,17 +530,22 @@
     }
 
     &:hover,
-    &.is-active,
     &[aria-current='page'],
     &[aria-current='true'] {
-      background: $color-sidenav-item-background-highlight;
+      background: $color-sidenav-item-background-hover;
```

Comment:
`[aria-current=...]` mean the same as `.is-active`, so if `.is-active` was removed from this selector, so should be these two.

Reply:
But I think we should actually keep `.is-active` here, because it needs to have the same background as hover.

Reply:
addressed